docs/en/observability/synthetics-manage-monitors.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[synthetics-manage-monitors]]
+= Manage synthetic monitors
+
```

Comment:
I'm thinking of using tabs like we do in [Create a synthetic monitor](https://observability-docs_1925.docs-preview.app.elstc.co/guide/en/observability/master/synthetic-run-tests.html#monitor-use). I think that will help push users toward the Monitor Management option. I can add tabs once we finalize the language (just to avoid complicating the review process 😄).